### ISSUE TITLE
Building innovative services using the real-time USGS water services data.

### ISSUE THREAD

**Issue by OpenGlobe** [OpenGlobe/myamerica-devsummit#10]
See http://waterservices.usgs.gov/

**Comment by sethfitz** [comment]
If USGS doesn't support CORS yet (useful for browser-based apps), I built [watercors](https://github.com/mojodna/watercors) a few years ago to proxy it (as well as CA's Department of Water Resources gauges).

**Comment by OpenGlobe** [comment]
This is great!  Will you be attending the summit?  If so continuing to do something interesting here might really help build out this session!

**Comment by sethfitz** [comment]
Yup!  I'll probably be doing something with @danrademacher related to social media in parks, but water is of personal interest to me (whitewater kayaker ← is a pic from a private trip on the Grand Canyon (hey, lotteries!)) and NHD is my favorite dataset!
